test/compflags/kushal/llvm-incremental.bad

```diff
@@ -0,0 +1 @@
+Incremental compilation is not yet supported with LLVM
```

Comment:
Ah, okay.  The line ought to be:
`llvm-incremental.chpl: error: Incremental compilation is not yet supported with LLVM` (but I have to build llvm to verify that)